app/js/views/navigation-menu.jsx

```diff
@@ -20,19 +20,67 @@ export default class NavigationMenu extends React.Component {
   render() {
     let route = location.hash.substr(1).split('/').shift();
     let menuNodes = [
-      ['services', 'Home'],
-      ['themes', 'Themes'],
-      ['mr-fox', 'Mr. Fox']
+      {
+        id: 'services',
+        label: 'Home',
+        icon: (
+          <svg xmlns="http://www.w3.org/2000/svg" height="30" width="30">
+            <path d="M3 27h8.256v-9.39h7.488V27H27V11.435L15 3 3 11.435V27z"
+                  fill="#fff"/>
+          </svg>
```

Comment:
Yeah, or maybe we can leverage blend mode or something to simulate active state with original image? Some day in SMS we agreed with VD to use something like http://azasypkin.github.io/style-my-image/ to change image states with CSS only without second image, wondering if we can do the same now.